x-pack/filebeat/module/aws/_meta/docs.asciidoc

```diff
@@ -0,0 +1,32 @@
+[role="xpack"]
+
+:modulename: aws
+:has-dashboards: false
+
+== AWS module
+
+beta[]
+
+This is a module for aws logs. It supports reading s3 server access logs from S3.
+
+[float]
+==== `s3_server_access_log` fileset settings
```

Comment:
I'd try to avoid using underscores in fileset names if possible. What do you think about calling this one `s3access`? In any case I think that `log` in the name is a bit redundant, and possibly also `server`.
If you want to use the underscore to have a more clear separation of AWS services maybe `s3_access` is also ok.

Reply:
Sounds good, will change it to `s3access`!